root/relationship/header.tt

```diff
@@ -1,7 +1,6 @@
-[% MACRO generic_link(url, text) BLOCK %]<a href="[% url %]">[% text %]</a>[% END %]
```

Comment:
This file and `root/user/profile/layout.tt` had this macro. You can see on http://chirlu.mbsandbox.org/user/Psychotic%20&%20Q-riosity that the ampersand is correctly encoded as `&amp;` now in the `href`, whereas https://beta.musicbrainz.org/user/Psychotic%20%26%20Q-riosity has illegal HTML.